tests/foreman/cli/test_lifecycleenvironment.py

```diff
@@ -366,10 +213,11 @@ def test_positive_list_all_with_per_page(self):
             {'organization-id': self.org['id'], 'per-page': per_page_count}
         )
 
-        self.assertEqual(len(lifecycle_environments), self.lces_count)
+        assert len(lifecycle_environments) == self.lces_count
```

Comment:
Lai `test_positive_list_all_with_per_page` looks ok to me.

Reply:
Great!  Everything should be resolved now.